test/test_corpus.cc

```diff
@@ -498,6 +512,117 @@ TEST_P(ExpectationTest, PerPhaseTest) { // NOLINT
     TEST_COUT << "errors OK" << '\n';
 } // namespace sorbet::test
 
+TEST_P(WhitequarkParserTest, PerPhaseTest) { // NOLINT
```

Comment:
Do I read this PR right that this section is never tested in CI? I think that our `bazel` runner (`test/BUILD`) only creates test suites for `PosTests` and `LSPTests`.

It's fine if that's the case, I just wanted to make sure that I'm understanding that you didn't mean for these tests to be run yet.

Reply:
Yes, #1670 has the following code in `test/BUILD`:
```
pipeline_tests(
    "whitequark_parser_corpus",
    glob([
        "whitequark/**/*.rb",
        "whitequark/**/*.exp",
    ]),
    "WhitequarkParserTests",
    filter = "WhitequarkParserTests",
)
```

I was going to run the script and generate all these tests in a separate PR. Right now this code is not used